specification/security/resource-manager/Microsoft.Security/preview/2023-07-01-preview/applications.json

```diff
@@ -0,0 +1,326 @@
+      }
+    },
+    "/{scope}/providers/Microsoft.Security/applicationMappingRule/{ruleId}/applications": {
+      "get": {
+        "x-ms-examples": {
+          "Retrieve an application": {
+            "$ref": "./examples/Applications/ListApplicationsByRuleId_example.json"
+          }
+        },
+        "tags": [
+          "Application"
+        ],
+        "description": "Get a list of all relevant applications over a rule Id.",
+        "operationId": "Application_ListByRuleId",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/RuleId"
+          },
+          {
+            "$ref": "#/parameters/Scope"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ApplicationList"
+            }
+          },
+          "default": {
+            "description": "Error response that describes why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ApplicationList": {
+      "type": "object",
+      "description": "Page of a application list",
+      "properties": {
+        "value": {
+          "description": "Collection of applications in this page",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Application"
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The URI to fetch the next page"
+        }
+      }
+    },
+    "Application": {
+      "type": "object",
+      "description": "The application business criticality resource",
+      "properties": {
+        "systemData": {
+          "readOnly": true,
+          "type": "object",
+          "description": "Azure Resource Manager metadata containing createdBy and modifiedBy information.",
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData"
+        },
+        "properties": {
+          "description": "Properties of an application business criticality",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ApplicationProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
+        }
+      ]
+    },
+    "ApplicationProperties": {
+      "type": "object",
+      "description": "Describes properties of the application",
+      "properties": {
+        "tenantId": {
+          "type": "string",
+          "description": "The tenant Id associated with the application (GUID)"
+        },
+        "environmentId": {
```

Comment:
From the example, this field is alawys equal to the scope, is it duplicated with the scope? Or is there other use case?